RepresentsMe/ViewControllers/Settings/AddressSettingsViewController.swift

```diff
@@ -67,6 +70,17 @@ class AddressSettingsViewController: UIViewController,
         stateTextField.text = state
     }
 
+    func didSelectLocation(location: CLLocationCoordinate2D, address: Address) {
+        setAddress(address)
+    }
+
+    func setAddress(_ address:Address) {
```

Comment:
Doesn't look like `AddressSettingsViewController#setAddress()` is being used anywhere else (unless I missed something), can we inline it to `AddressSettingsViewController#didSelectLocation()`